packages/admin/admin/src/error/errorboundary/ErrorBoundary.tsx

```diff
@@ -1,4 +1,5 @@
 import { ChevronDown, ChevronRight, Error } from "@comet/admin-icons";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
@thomasdax98 IMO the Error Boundary shouldn't use the MUI Alert. I've tried using our own Alert component, but that doesn't look right. Maybe we can discuss this with UX?

Reply:
Alright, I'll take a look at that